src/components/atoms/VolumeControl/VolumeControl.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback } from "react";
+import { faVolumeMute, faVolumeUp } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+
+import {
+  DEFAULT_AUDIO_VOLUME,
+  MAX_AUDIO_VOLUME,
+  MIN_AUDIO_VOLUME,
+} from "settings";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./VolumeControl.scss";
+
+export interface VolumeControlProps {
+  className?: string;
+  label?: string;
+  name: string;
+  muted?: boolean;
+  onMute?: () => void;
+  onVolume?: (volume: number) => void;
+  volume?: number;
+  withMute?: boolean;
+  withSlider?: boolean;
+}
+
+export const VolumeControl: React.FC<VolumeControlProps> = ({
+  className = "",
+  label = "",
```

Comment:
do we have to set them? 
```suggestion
  className,
  label,
```

Reply:
Do we have to not set them?

Reply:
I left them as they are. It's simple to see defaults at the top before digging into the meat of the component